catalog/spatial/geocoding/spatial-geocoding-api/src/main/java/org/codice/ddf/spatial/geocoding/GeoEntryMetacardType.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.geocoding;
+
+import com.google.common.collect.ImmutableSet;
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import java.util.Set;
+
+public class GeoEntryMetacardType implements MetacardType {
+
+  public static final String FEATURE_CODE_ATTRIBUTE_NAME = "ext.feature-code";
+
+  public static final String FEATURE_CLASS_ATTRIBUTE_NAME = "ext.feature-class";
+
+  public static final String POPULATION_ATTRIBUTE_NAME = "ext.population";
+
+  private static final String METACARD_TYPENAME = "GeoEntryMetacardType";
+
+  private static final Set<AttributeDescriptor> DESCRIPTORS =
```

Comment:
Should this also include CoreAttributes?  And LocationAttributes?

Reply:
This is a part of Ben's PR that I built on top of. I'm not sure about that.